boards/arm/ti_am62x_sk_m4/doc/index.rst

```diff
@@ -0,0 +1,74 @@
+.. _ti_am62x_sk_m4:
+
+AM62x-SK M4F Core
+#################
+
+Overview
+********
+
+The AM62x-SK board configuration is used by Zephyr applications that run on
+the TI AM62x platform. The board configuration provides support for the ARM
+Cortex-M4F CPU and the following devices:
+
+- Nested Vector Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+
+The board configuration also enables support for the semihosting debugging console.
+
+See the `TI AM62X Product Page`_ for details.
+
+Hardware
+********
+
```

Comment:
Please add a line here something on the lines of:

The AM62x SoC has 1 MCU domain with the following specs:

Reply:
Have added the line now.